grouper-misc/grouper-authentication/plugin-pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2020 Internet2
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<!-- This POM specifies which jars are included in the bundle.
+     This artifact is not stored in maven central (for space reasons), and is only used in container
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>edu.internet2.middleware.grouper</groupId>
+        <artifactId>grouper-parent</artifactId>
+        <version>2.6.0-SNAPSHOT</version>
+        <relativePath>../../grouper-parent</relativePath>
+    </parent>
+
+    <name>Grouper Authentication</name>
+    <description>Authentication Library</description>
+    <artifactId>grouper-authentication</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <packaging>bundle</packaging>
+
+    <properties>
+        <pac4j.version>4.3.1</pac4j.version>
+        <jee-pac4j.version>5.0.0</jee-pac4j.version>
+        <grouper.version>2.6.0-SNAPSHOT</grouper.version>
+    </properties>
+
+    <dependencies>
+    
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>grouper-ui</artifactId>
+            <version>${grouper.version}</version>
+            <scope>provided</scope>
```

Comment:
This might need to be removed. Since this is a different classloader, this stuff probably needs to be packaged in with the plugin